### DISCUSSION TITLE
Light interference

### DISCUSSION THREAD

**Discussion by marcoromelli** [mitsuba-renderer/mitsuba3#1414] [Feature requests] [answered]
Hello! Is there any plan to add support for light interference in Mitsuba? 
I'm not expert enough to be sure but it seems possible in a similar way as it was done for polarization and would open Mitsuba to fields like interferometry.

**Comment by rtabbara** [comment, marked as answer]
Hi @marcoromelli ,

As far as I'm aware, we currently have no plans to support light interference in Mitsuba 3
